src/transformers/models/prophetnet/modeling_prophetnet.py

```
@@ -762,22 +756,17 @@ def forward(
             p=self.attention_dropout,
             training=self.training,
         )
-
-        attn_output = torch.bmm(attn_probs, value_states)
+        attn_output = torch.einsum("bsij,bsjk->bsik", attn_probs, value_states)
         assert attn_output.size() == (
```

same here, let's raise an error